src/main/java/com/urbanairship/api/templates/model/TemplateScheduledPushPayload.java

```diff
@@ -0,0 +1,167 @@
+package com.urbanairship.api.templates.model;
+
+
+import com.google.common.base.Preconditions;
+import com.urbanairship.api.push.model.DeviceTypeData;
+import com.urbanairship.api.push.model.audience.Selector;
+import com.urbanairship.api.schedule.model.Schedule;
+
+public class TemplateScheduledPushPayload {
+    private final Selector audience;
```

Comment:
An Optional name field can be set in the push payload so we should try and add support for that as well.
https://docs.airship.com/api/ua/#operation/api/templates/schedules/post/requestbody/items/name

Reply:
done